storageservice/modules/blobs/Module.md

```diff
@@ -113,78 +43,68 @@ If you are using one of the Access Key, use the follwing format.
 
 ```ballerina
     azure_blobs:AzureBlobServiceConfiguration blobServiceConfig = {
-        accessKeyOrSAS: os:getEnv("ACCESS_KEY_OR_SAS"),
-        accountName: os:getEnv("ACCOUNT_NAME"),
+        accessKeyOrSAS: "ACCESS_KEY_OR_SAS",
+        accountName: "ACCOUNT_NAME",
         authorizationMethod: "accessKey"
     };
 ```
 
-## Step3: Initialize Azure Storage Blob Client 
-
-Create the BlobClient using the configuration you have created as shown above.
+Create the BlobClient using the blobServiceConfig you have created as shown above.
```

Comment:
Shall we make this and the above point under bullet points?

Reply:
Done